src-tauri/Cargo.toml

```diff
@@ -17,7 +17,7 @@ tauri-build = { version = "1.5.0", features = [] }
 [dependencies]
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
-tauri = { version = "1.5.3", features = ["dialog-message"] }
+tauri = { version = "1.5.3", features = [ "shell-open", "dialog-message"] }
```

Comment:
I wouldn't go this route, I think we can opt to open a window and load that page similar to what we do with the local run ? what do you think 🤔

Reply:
To me, opening Grafana in the browser makes more sense. Here are some of the reasons I can think of right now:
* You probably have your Grafana credentials saved in the browser, so if you have to re-login, it's easier to do in the browser
* It makes sharing the link easier
* If the user opens the chart in Explore or decided to export results as PDF, a new tab is opened where they are directed to a different place in Grafana/app.k6.io - will this work and look okay-ish in a desktop window?

Reply:
I think there are multiple ways to tackle those things (after all we do have the credentials for tests!) but that overall falls into the cloud integration category that is not a priority for this week beside running the test. 
So I'm convinced this is the best approach for now 👍